pendulum_control/test/test_pendulum_demo.py.in

```diff
@@ -0,0 +1,68 @@
+import os
+from launch import LaunchDescriptor
+from launch.exit_handler import ignore_exit_handler
+from launch.launcher import DefaultLauncher
+from launch_testing import create_handler
+
+
+def setup():
+    os.environ['OSPL_VERBOSITY'] = '8'  # 8 = OS_NONE
+
+
+def test_executable():
+    output_handlers = []
+
+    launch_descriptor = LaunchDescriptor()
+
+    pendulum_logger_executable = '@RCLCPP_DEMO_PENDULUM_LOGGER_EXECUTABLE@'
+    pendulum_logger_output_file = '@RCLCPP_DEMO_PENDULUM_LOGGER_EXPECTED_OUTPUT@'
+    pendulum_logger_name = 'test_executable_0'
+    pendulum_logger_handler = create_handler(pendulum_logger_name, launch_descriptor, pendulum_logger_output_file)
+    assert pendulum_logger_handler, 'Cannot find appropriate handler for %s' % pendulum_logger_output_file
+    output_handlers.append(pendulum_logger_handler)
+    launch_descriptor.add_process(
+        cmd=[pendulum_logger_executable],
+        name=pendulum_logger_name,
+        exit_handler=ignore_exit_handler,
+        output_handlers=[pendulum_logger_handler],
+    )
+
+    pendulum_demo_executable = '@RCLCPP_DEMO_PENDULUM_DEMO_EXECUTABLE@'
+    pendulum_demo_output_file = '@RCLCPP_DEMO_PENDULUM_DEMO_EXPECTED_OUTPUT@'
+    pendulum_demo_name = 'test_executable_1'
+    pendulum_demo_handler = create_handler(pendulum_demo_name, launch_descriptor, pendulum_demo_output_file)
+    assert pendulum_demo_handler, 'Cannot find appropriate handler for %s' % pendulum_demo_output_file
+    output_handlers.append(pendulum_demo_handler)
+    launch_descriptor.add_process(
+        cmd=[pendulum_demo_executable, '-i', '0'],
```

Comment:
I'm not sure how launch works, so bear with me--specifying `-i 0` will make the executable run indefinitely until interrupted. When will the pendulum_demo executable get interrupted?